### PR TITLE
Feature/uia imagesview tests

### PR DESCRIPTION
**Pull request by nlesniewski**

### CODE REVIEW

File: app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java [hunk 1]
```diff
@@ -1,16 +1,100 @@
...
+
+import static org.junit.Assert.assertTrue;
 
 public class GoInvisiblePackage extends UiView {
 
+    @SuppressWarnings("SpellCheckingInspection")
+    private static final String GOINVISIBLE_PACKAGE_TEST = "com.invisibleteam.goinvisible.test";
+    private static final String OUTPUT_PATH = Environment.getExternalStorageDirectory().getPath() + "/Pictures";
+
     public static void launch() {
```
Comment (BartSoftware) [inline app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java]: launch is to less for me, maybe launchApp or startApp?

File: app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java [hunk 2]
```diff
@@ -1,16 +1,100 @@
...
+import com.invisibleteam.goinvisible.pages.system.SystemHomeView;
+import com.invisibleteam.goinvisible.utilities.ShellCmdHelper;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertTrue;
 
 public class GoInvisiblePackage extends UiView {
```
Comment (BartSoftware) [inline app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java]: is this heritage necessary?

**Review by BartSoftware — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java (hunk 1 above)
Comment (nlesniewski) [reply app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java]: No idea what else can we launch.

App is redundant.

File: app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java (hunk 2 above)
Comment (nlesniewski) [reply app/src/androidTest/java/com/invisibleteam/goinvisible/pages/GoInvisiblePackage.java]: No idea what are You talking about.

**Pull request merged**